src/backend/controllers/userController.js

```diff
@@ -232,7 +232,8 @@ exports.login = async (req: Request, res: Response) => {
 		const payload = {
 			user: {
 				// Primary key id generated by MongoDB
-				id: user.id
+				id: user.id,
+				issued: Date.now()
```

Comment:
The issue here is that there is the small possibility that the `issued` property here and the `lastLogin` in the user might be _ever so slightly_ off, since they call separate instances of `Date.now()`

So before the logic that checks if you've logged in before, do a singular `const now = Date.now()` and then set everything that needs that `Date.now()` to `now`